wasm/src/programs/verifying_key/credits.rs

```diff
@@ -26,249 +26,264 @@ impl VerifyingKey {
         VerifyingKey::from_bytes(&snarkvm_parameters::testnet::BondPublicVerifier::load_bytes().unwrap()).unwrap()
     }
 
+    /// Returns the verifying key for the bond_validator function
+    ///
+    /// @returns {VerifyingKey} Verifying key for the bond_validator function
+    #[wasm_bindgen(js_name = "bondValidatorVerifier")]
+    pub fn bond_validator_verifier() -> VerifyingKey {
+        let vk = CurrentNetwork::get_credits_verifying_key("bond_validator".to_string()).unwrap().clone();
+        let num_variables = vk.circuit_info.num_public_and_private_variables as u64;
+        VerifyingKey::from(VerifyingKeyNative::new(vk, num_variables))
+    }
+
     /// Returns the verifying key for the claim_delegator function
     ///
     /// @returns {VerifyingKey} Verifying key for the claim_unbond_public function
     #[wasm_bindgen(js_name = "claimUnbondPublicVerifier")]
     pub fn claim_unbond_public_verifier() -> VerifyingKey {
-        VerifyingKey::from_bytes(&snarkvm_parameters::testnet::ClaimUnbondPublicVerifier::load_bytes().unwrap())
-            .unwrap()
+        let vk = CurrentNetwork::get_credits_verifying_key("bond_validator".to_string()).unwrap().clone();
```

Comment:
Why is it now using hardcoded strings, whereas before it used methods (which are statically typed and safer)?

Reply:
Because the VerifyingKeys in SnarkVM are for different object than the verifying key (something called the CircuitVerifyingKey, which VerifyingKey is based off of). Loading it directly from bytes fails (feel free to try it and confirm).

However this method requires the actual VerifyingKey object to be accepted for Executions and thus this uses the convenience method in the Network object to do this.